tests/workflow/eden.networking.tests.txt

```diff
@@ -0,0 +1,67 @@
+# Number of tests
+{{$tests := 10}}
+k EDEN_TEST_SETUP env. var. -- "y"(default) performs the EDEN setup steps
+{{$setup := "y"}}
+{{$setup_env := EdenGetEnv "EDEN_TEST_SETUP"}}
+{{if $setup_env}}{{$setup = $setup_env}}{{end}}
+# EDEN_TEST_STOP -- "y" stops EDEN after tests ("n" by default)
+{{$stop := EdenGetEnv "EDEN_TEST_STOP"}}
+# EDEN_TEST_REGISTRY env. var. -- "y"(default) performs the local EDEN registry test
+{{$registry := EdenGetEnv "EDEN_TEST_REGISTRY"}}
+
+{{$devmodel := EdenConfig "eve.devmodel"}}
+
+{{if (ne $setup "n")}}
+#./eden config add default
+/bin/echo Eden setup (1/{{$tests}})
+eden.escript.test -test.run TestEdenScripts/eden_setup
+#source ~/.eden/activate.sh
+{{end}}
+
+{{if or (eq $devmodel "ZedVirtual-4G") (eq $devmodel "VBox") (eq $devmodel "parallels") }}
+eden+ports.sh 2223:2223 2224:2224 5912:5902 5911:5901 8027:8027 8028:8028 8029:8029 8030:8030 8031:8031
+{{end}}
+
+{{if (ne $setup "n")}}
+/bin/echo Eden start (2/{{$tests}})
+eden.escript.test -test.run TestEdenScripts/eden_start
+
+/bin/echo Eden onboard (3/{{$tests}})
+eden.escript.test -test.run TestEdenScripts/eden_onboard
+/bin/echo Eden template check (3.1/{{$tests}})
+eden.escript.test -test.run TestEdenScripts/template_check
+{{end}}
+
+/bin/echo Eden basic network test (4/{{$tests}})
+eden.escript.test -testdata ../network/testdata/ -test.run TestEdenScripts/network_test
+/bin/echo Eden basic VLAN test (4.1/{{$tests}})
+eden.escript.test -testdata ../network/testdata/ -test.run TestEdenScripts/switch_net_vlans
+/bin/echo Eden basic VLAN test (4.2/{{$tests}})
+eden.escript.test -testdata ../network/testdata/ -test.run TestEdenScripts/vlans_and_bonds
+
+/bin/echo Eden ACL to particular host (5.1/{{$tests}})
+eden.escript.test -testdata ../eclient/testdata/ -test.run TestEdenScripts/acl
+/bin/echo Eden Network light (5.2/{{$tests}})
+eden.escript.test -testdata ../eclient/testdata/ -test.run TestEdenScripts/networking_light
+
+/bin/echo Eden Networks switch (6.1/{{$tests}})
+eden.escript.test -testdata ../eclient/testdata/ -test.run TestEdenScripts/nw_switch
+/bin/echo Eden Network Ports switch (6.2/{{$tests}})
+eden.escript.test -testdata ../eclient/testdata/ -test.run TestEdenScripts/port_switch
+/bin/echo Eden Network portmap test (6.3/{{$tests}})
+eden.escript.test -testdata ../eclient/testdata/ -test.run TestEdenScripts/port_forward
+/bin/echo Eden test app info (6.4/{{$tests}})
+eden.escript.test -testdata ../eclient/testdata/ -test.run TestEdenScripts/app_dhcp
+
+/bin/echo Eden VNC (7/{{$tests}})
+eden.escript.test -testdata ../vnc/testdata/ -test.run TestEdenScripts/vnc_test
+
+/bin/echo Eden Networking via switch test (8/{{$tests}})
+eden.escript.test -testdata ../eclient/testdata/ -test.run TestEdenScripts/air-gapped-switch
+
+/bin/echo Eden Nginx (9/{{$tests}})
+eden.escript.test -testdata ../eclient/testdata/ -test.run TestEdenScripts/ngnix
```

Comment:
Some of these tests were not enabled (i.e. part of the default workflow) for quite some time so they could be failing, we will see...